python_modules/libraries/dagster-mysql/dagster_mysql/event_log/event_log.py

```diff
@@ -208,12 +205,6 @@ def watch(self, run_id: str, cursor: Optional[str], callback: EventHandlerFn) ->
     def end_watch(self, run_id: str, handler: EventHandlerFn) -> None:
         self._event_watcher.unwatch_run(run_id, handler)
 
-    @property
-    def supports_intersect(self) -> bool:
-        return parse_mysql_version(self._mysql_version) >= parse_mysql_version(  # type: ignore  # (possible none)
-            MINIMUM_MYSQL_INTERSECT_VERSION
```

Comment:
Another option here is to change this to the version range that we know works 🤷 

But I'm not aware of any performance benchmarking we've done to show `INTERSECT` is better when available - probably not worth maintaining the complexity.